File: c#/program23.cs
using System;

class NumberX
{
    internal int iPower;
    internal int iBase;

    internal NumberX(int no1, int no2)
    {
        this.iBase = no1;
        this.iPower = no2;
    }

    public int CalculatePower()
    {
        int Result = 1;

        for(int i = 1; i <= iPower; i++)
        {
            Result = Result * iBase;
        }
        return Result;
    }
}

class program23
{
    public static void Main()
    {
        int value1 = 0, value2 = 0;

        Console.WriteLine("Enter Base :");
        value1 = int.Parse(Console.ReadLine());

        Console.WriteLine("Enter Power :");
        value2 = int.Parse(Console.ReadLine());

        NumberX obj = new NumberX(value1, value2);
        Console.WriteLine("Result is : "+obj.CalculatePower());
    }
}

File: java/program23.java
import java.util.*;

class Number 
{
    public int iBase;
    public int iPower;

    public Number(int no1,int no2)
    {
        iBase = no1;
        iPower = no2;
    }

    public int CalculatePower()
    {
        int iResult = 1;

        for(int iCnt = 1; iCnt <= iPower; iCnt++)
        {
            iResult = iResult * iBase;
        }
        return iResult;
    }
}

class program23
{
    public static void main(String arg[])
    {
        int iValue1 = 0;
        int iValue2 = 0,iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Base :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter power :");
        iValue2 = sobj.nextInt();

        Number obj = new Number(iValue1,iValue2);
        iRet = obj.CalculatePower();

        System.out.println("Result is : "+iRet);
    }
}